master/internal/resourcemanagers/agent_resource_manager.go

```diff
@@ -167,8 +167,8 @@ func (a *agentResourceManager) Receive(ctx *actor.Context) error {
 				ctx.Log().WithError(aMsg).Error("")
 				ctx.Respond(aMsg)
```

Comment:
if there are multiple errors we will ctx.Respond multiple times, but a ctx.Respond closes the channel it sends on and sends on close channels panic, so this will crash the rm if a resource pool crashes.